extra-dev/packages/coq-hott/coq-hott.8.10.dev/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "Jason Gross <jgross@mit.edu>"
+homepage: "http://homotopytypetheory.org/"
+bug-reports: "https://github.com/HoTT/HoTT/issues"
+license: "BSD 2-clause"
+build: [
+  ["bash" "-c" "./autogen.sh -skip-submodules || :"]
```

Comment:
```suggestion
  ["sh" "-c" "./autogen.sh -skip-submodules || :"]
```
Just checking if this really depends on `bash` (which may not be installed) or if regular `sh` would suffice.

Reply:
I'm not sure.  Does `if [ "$1" != "-skip-submodules" ] && command -v git >/dev/null 2>&1` need `bash`?  What about `trap "exit 1" SIGINT SIGTERM`.  (https://github.com/HoTT/HoTT/blob/master/autogen.sh is a bit complicated, but I think those are the main things)

Reply:
OK, that seems much more complicated than I hoped it would be, let's keep the `bash` there and I'll merge as soon as CI passes.